etcdctl/ctlv3/command/member_command.go

```diff
@@ -111,8 +112,34 @@ func memberAddCommandFunc(cmd *cobra.Command, args []string) {
 	if err != nil {
 		ExitWithError(ExitError, err)
 	}
+	newID := resp.Member.ID
 
 	display.MemberAdd(*resp)
+
+	ctx, cancel = commandCtx(cmd)
+	listResp, err := mustClientFromCmd(cmd).MemberList(ctx)
+	cancel()
+
+	if err != nil {
+		ExitWithError(ExitError, err)
+	}
+
+	conf := []string{}
+	for _, memb := range listResp.Members {
+		for _, u := range memb.PeerURLs {
+			n := memb.Name
+			if memb.ID == newID {
+				n = newMemberName
+			}
+			conf = append(conf, fmt.Sprintf("%s=%s", n, u))
+		}
+	}
+
+	fmt.Print("\n")
+	fmt.Printf("ETCD_NAME=%q\n", newMemberName)
+	fmt.Printf("ETCD_INITIAL_CLUSTER=%q\n", strings.Join(conf, ","))
+	fmt.Printf("ETCD_INITIAL_CLUSTER_STATE=\"existing\"\n")
```

Comment:
No. This shouldn't go in `printer` since those functions are supposed to map directly to RPCs. Adding the member list stuff turns the command into a composite command. The way to support additional information is similar to the way it's done in get_command.go:
```go
if _, isSimple := (display).(*simplePrinter); !isSimple {
// fetch list
// print ETCD variables
}
```

The README.md also needs to be updated to reflect the change in the etcdctl output.